docs/operations/hotplug.md

```diff
@@ -0,0 +1,129 @@
+# Hotplug Volumes
```

Comment:
Can you change the filename to match H1?  This helps keeping the file hierarchy from becoming spaghetti.  Thanks!

Reply:
Yes